src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -419,7 +419,8 @@ def load_arguments(self, _):
         c.extra('cmd')
 
     with self.argument_context('network express-route peering') as c:
-        c.argument('peer_asn', help='Autonomous system number of the customer/connectivity provider.')
+        # TODO 'type=long'? - https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-network/azure/mgmt/network/v2017_11_01/models/express_route_circuit_peering.py#L86
```

Comment:
"long" is a valid Py2 type (has been removed in Py3)

Reply:
Which 'type' would I put in the argument registration on the line below to support both Python 2 and 3?
Since long doesn't exist in Py3, I just went with int.

Reply:
Also, not sure how this works currently...
Seems like Autorest 3 does more validation?

Reply:
No, this didn't change and is here from day one.
What the purpose of the "type" parameter in the CLI?
If it's help only, it's fine. If it's validation, might be a problem.
You can use six if you need an accurate type based on Python version:
https://pythonhosted.org/six/#six.integer_types